services/maven-metadata/maven-metadata-redirect.service.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { redirector } = require('..')
+
+module.exports = [
+  redirector({
+    category: 'version',
+    route: {
+      base: 'maven-metadata/v',
+      pattern: ':protocol(http|https)/:hostAndPath(.*?)',
```

Comment:
I believe the lazy pattern for `hostAndPath` is necessary to prevent it from sucking in an extension format (`.json`, `.svg`) if one is specified. Let me know if that's not the case though

Reply:
Hmm, is that a limitation specific to redirects?

Reply:
> Hmm, is that a limitation specific to redirects?

I think it may be. I traced it to this point and found that the `.json` extension from the tests was getting pulled into the `hostAndPath` match so this was always injecting an extra `.svg`. Gave up after a while though and just changed the pattern 😄 

https://github.com/badges/shields/blob/9fe4d739b0071ec960f4a4704f1b63c82c1fee6f/core/base-service/redirector.js#L113-L116

Reply:
I took another look at this, I switched it `:hostAndPath+` and everything is working as expected. When I first created the redirector I copied in the pattern from the service which had `:hostAndPath(.+)` which wasn't working